hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/auth/delegation/DelegationConstants.java

```diff
@@ -160,6 +160,18 @@ public final class DelegationConstants {
    */
   public static final String STS_STANDARD = "sts.amazonaws.com";
 
+  /**
+   * The format of the STS Endpoint
+   */
```

Comment:
you can move both of these into STSClientFactory as they are only relevant for that class.